src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -147,14 +147,23 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    action='store_true',
                    arg_group='Basic Policy Options',
                    help='Use open restriction. License or key will be delivered on every request.')
+        c.argument('symmetric',
+                   action='store_true',
+                   arg_group='Token Restriction Key Type',
+                   help='Use a symmetric token key.')
+        c.argument('rsa',
+                   action='store_true',
+                   arg_group='Token Restriction Key Type',
+                   help='Use an RSA certificate as a token key.')
+        c.argument('x509',
+                   action='store_true',
+                   arg_group='Token Restriction Key Type',
+                   help='Use a certificate in PKCS12 format (X509Certificate2 in .NET) as a token key.')
         c.argument('policy_option_name', help='The content key policy option name.')
         c.argument('policy_option_id', help='The content key policy option identifier.')
         c.argument('issuer', arg_group='Token Restriction', help='The token issuer.')
         c.argument('audience', arg_group='Token Restriction', help='The audience for the token.')
-        c.argument('symmetric_token_key', arg_group='Token Restriction', help='The key value of the key.')
-        c.argument('rsa_token_key_exponent', arg_group='Token Restriction', help='The RSA Parameter exponent.')
-        c.argument('rsa_token_key_modulus', arg_group='Token Restriction', help='The RSA Parameter modulus.')
-        c.argument('x509_certificate_token_key', arg_group='Token Restriction', help='The raw data field of a certificate in PKCS 12 format (X509Certificate2 in .NET) with \\n as newlines')
+        c.argument('token_key', arg_group='Token Restriction', help='Either a string (for symmetric key) or a filepath to a certificate (x509, rsa). Must be used with a Token Restriction Key Type flag.')
```

Comment:
I think it would be better 'the local full path to a certificate' to be consistent with what's already done.